src/unix/haiku.c

```diff
@@ -0,0 +1,174 @@
+/* Copyright Joyent, Inc. and other Node contributors. All rights reserved.
```

Comment:
Can you use this copyright instead: https://github.com/libuv/libuv/blob/ce7f9c889e2afb7791f6385471abc9032ac91ff5/src/unix/ibmi.c#L1-L20